cluster-state-service/internal/features/e2e_list_tasks.feature

```diff
@@ -54,6 +54,16 @@ Feature: List Tasks
       | count |
       |   1   |
 
+  Scenario: List tasks with filter combination
+    Given I start <count> task in the ECS cluster
+    When I try to list tasks in the ECS cluster with status running and startedBy someone filters
```

Comment:
The convention we're trying to follow if to  use `try to` when we expect an error. Lets change this to `When I list tasks in the ECS cluster with status running and startedBy someone filters`